libs/experimental/pyproject.toml

```diff
@@ -15,8 +15,6 @@ langchain = ">=0.0.350,<0.1"
 presidio-anonymizer = {version = "^2.2.33", optional = true}
 presidio-analyzer = {version = "^2.2.33", optional = true}
 faker = {version = "^19.3.1", optional = true}
-vowpal-wabbit-next = {version = "0.6.0", optional = true}
-sentence-transformers = {version = "^2", optional = true}
```

Comment:
these have issues in 3.12. Functionality will still work in compatible environments, but associated unit tests won't run in CI. Should discuss.